README.md

```diff
@@ -55,10 +55,26 @@ curl -L http://127.0.0.1:4001/v2/keys/mykey -XPUT -d value="this is awesome"
 curl -L http://127.0.0.1:4001/v2/keys/mykey
 ```
 
-You have successfully started an etcd on a single machine and written a key to the store. Now it's time to dig into the full etcd API and other guides.
+You have successfully started an etcd and written a key to the store.
+
+### Running local etcd cluster
+
+First install [goreman](https://github.com/mattn/goreman), which manages Procfile-based applications.
+
+We write [a Procfile script](./Procfile) which contains an local example cluster, and you can start it:
```

Comment:
What about:

Our [Profile script]() will set up a local example cluster. You can start it with: